docs/content/engines/hive.md

```diff
@@ -211,6 +211,18 @@ OK
 3	Paimon
 */
 
+```
+## Setting of location Location 
+In addition to the way setting location above, you can also place the location setting in TBProperties 
+to avoid Hive accessing Paimon's location through its own file system when creating tables.
+This method is effective in scenarios using Object storage,such as s3.
```

Comment:
For the object storage, why does this need to configure the `paimon_meta_location`? IMO, using `location` is enough to satisfy the object storage need, otherwise the filesystem and the metastore of Hive will lead to inconsistencies.

Reply:
Hi~ @SteNicholas  thanks for your concern.
When using `location`, Hive will use its own FileSystem for authentication and to create the table path. This bypasses Paimon's fileio configuration, causing authentication errors, especially in object storage scenarios.